cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -85,13 +71,13 @@ bool FliImpl::isValueLogic(mtiTypeIdT type)
     return false;
 }
 
-bool FliImpl::isValueChar(mtiTypeIdT type)
+bool fli_is_char(mtiTypeIdT type)
```

Comment:
Is there any reason these actually need to be changed from the previous member functions to free functions?

I'm not opposed to changing them, I'm just trying to follow if this is unrelated tidy-up. If so, this too would make sense as a small standalone PR.

Remember it's not just about making the review easy - it's about giving people looking at the blame some idea of why things were changed.

Reply:
This is just some clean-up.  It was on separate commit prior to the request to squash, but given the state now, I am not looking to change unless there is compelling reason to do so.